.gitignore

```diff
@@ -0,0 +1,212 @@
+!packages/*/build/
+
+# MSTest test Results
+[Tt]est[Rr]esult*/
+[Bb]uild[Ll]og.*
+
+*_i.c
+*_p.c
+*.ilk
+*.meta
+*.obj
+*.pch
+*.pdb
+*.pgc
+*.pgd
+*.rsp
+*.sbr
+*.tlb
+*.tli
+*.tlh
+*.tmp
+*.tmp_proj
+*.log
+*.vspscc
+*.vssscc
+.builds
+*.pidb
+*.log
+*.scc
+*.bak
+
+# Visual C++ cache files
+ipch/
+*.aps
+*.ncb
+*.opensdf
+*.sdf
+*.cachefile
+
+# Visual Studio profiler
+*.psess
+*.vsp
+*.vspx
+
+# Guidance Automation Toolkit
+*.gpState
+
+# ReSharper is a .NET coding add-in
+_ReSharper*/
+*.[Rr]e[Ss]harper
+
+# TeamCity is a build add-in
+_TeamCity*
+
+# DotCover is a Code Coverage Tool
+*.dotCover
+
+# NCrunch
+*.ncrunch*
+.*crunch*.local.xml
+
+# Installshield output folder
+[Ee]xpress/
+
+# DocProject is a documentation generator add-in
+DocProject/buildhelp/
+DocProject/Help/*.HxT
+DocProject/Help/*.HxC
+DocProject/Help/*.hhc
+DocProject/Help/*.hhk
+DocProject/Help/*.hhp
+DocProject/Help/Html2
+DocProject/Help/html
+
+# Click-Once directory
+publish/
+
+# Publish Web Output
+*.Publish.xml
+*.pubxml
+
+# NuGet Packages Directory
+## TODO: If you have NuGet Package Restore enabled, uncomment the next line
+packages/
+
+# Windows Azure Build Output
+csx
+*.build.csdef
+
+# Windows Store app package directory
+AppPackages/
+
+# Others
+sql/
+*.Cache
+ClientBin/
+[Ss]tyle[Cc]op.*
+~$*
+*~
+*.dbmdl
+*.[Pp]ublish.xml
+*.pfx
+*.publishsettings
+
+# RIA/Silverlight projects
+Generated_Code/
+
+# Backup & report files from converting an old project file to a newer
+# Visual Studio version. Backup files are not needed, because we have git ;-)
+_UpgradeReport_Files/
+Backup*/
+UpgradeLog*.XML
+UpgradeLog*.htm
+
+# SQL Server files
+App_Data/*.mdf
+App_Data/*.ldf
+
+# =========================
+# Windows detritus
+# =========================
+
+# Windows image file caches
+Thumbs.db
+ehthumbs.db
+
+# Folder config file
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Mac crap
+.DS_Store
+
+# =========================
+# Orchard specifics
+# =========================
+
+App_Data/
+glob:*.user
+*.patch
+*.hg
+build/
+/buildazure
+/buildtasks
+/artifacts
+*.sln.cache
+src/Orchard.Web/Media/*
+log.xml
+profiling/
+*.orig
+src/Orchard.Web/Modules-temp/*
+src/Backup/*
+src/packages/*
+src/UpgradeLog.*
+*.itrace.csdef
+*.build.csdef
+src/Orchard.Azure/Orchard.Azure.Web/Core/
+src/Orchard.Azure/Orchard.Azure.Web/Modules/
+src/Orchard.Azure/Orchard.Azure.Web/Themes/
+src/Orchard.Azure/Orchard.Azure.CloudService/csx/Debug/*
+src/Orchard.Web/Orchard.Web.Publish.xml
+src/TestResults/*
+src/Orchard.Web/Properties/PublishProfiles
+src/Orchard.Azure/Orchard.Azure.CloudService/Staging/
+src/Orchard.Web/Modules/Orchard.Azure.MediaServices/node_modules
+src/Orchard.Web/Modules/Orchard.Layouts/node_modules
+src/Orchard.Web/Modules/Orchard.DynamicForms/node_modules
+
+#enable all /lib artifacts
+!lib/*/*.*
+
+# =========================
+# devofficecom specifics
```

Comment:
Ok. I will update the gitignore file and remove unnecessary files as we discussed.